### ISSUE TITLE
Find shorter prefixes in the assembly disambiguator

### ISSUE THREAD

**Issue by chriseth** [powdr-labs/powdr#304]
Disambiguated names get rather long, we should find much shorter prefixes.

**Comment by leonardoalt** [comment]
You mean specifically for riscv?

**Comment by chriseth** [comment]
Yes, for riscv, but we will use the same code for delendum as well - essentially anything that uses llvm.
